Formula/libmf.rb

```diff
@@ -0,0 +1,41 @@
+class Libmf < Formula
+  desc "open source tool for approximating an incomplete matrix."
+  homepage "https://www.csie.ntu.edu.tw/~cjlin/libmf/"
+  url "http://www.csie.ntu.edu.tw/~cjlin/libmf/libmf-2.01.zip"
+  sha256 "75bb98a4e8f1a318d7d07556583727b4e301759904570bf527972d964d34ef30"
+  patch :DATA
+
+  def install
+    system "make"
+    system "make", "lib"
+    cp "./mf-predict", bin
```

Comment:
Use `bin.install`. Also; can you ask upstream if they would consider a `make install`.